types/jest/index.d.ts

```diff
@@ -29,7 +29,7 @@
 //                 Regev Brody <https://github.com/regevbr>
 //                 Mark Skelton <https://github.com/mskelton>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.8
```

Comment:
I think `TypeScript Version` can be 3.2, is not it?

Reply:
No, because the dependencies use `import type`, which is only supported [since **TypeScript 3.4**](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-exports).

Reply:
@ExE-Boss is that whats blocking this PR, or was it abandoned for another reason?

I've noticed in my dependency tree I've got some duplication caused by `@types/jest` being at the top of the tree: it causes some jest 25 packages to pulled in, which causes jest to duplicate itself in a bunch of places since I'm using 26.

I'm happy to do the dependency update, but wanted to check if this was the blocker before getting started :)

Reply:
Just recreated this PR:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47545